client/public/index.html

```diff
@@ -6,6 +6,10 @@
     <meta name="theme-color" content="#000000">
     <meta name="version" content="%REACT_APP_VERSION%">
 
+    <!--
+      Google Material UI icons
+    -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
It looks like this points to a CSS file that has a WOFF font in it that's a 100K download, are we comfortable with adding that to the site's page weight?  How many icons are we actually using?  If it's only like, one icon, we might want to just get its SVG and use that...

Reply:
Removing it entirely!